draft-ietf-privacypass-architecture.md

```diff
@@ -174,14 +132,69 @@ The following terms are used throughout this document.
   attested to by the Attester.
 - Attester: An entity that attests to properties of Client for the
   purposes of token issuance.
+- Redemption context: The interactions and set of information shared
+between the Client and Origin.
+- Issuance context: The interactions and set of information shared
+between the Client, Attester, and Issuer.
 
 # Architecture
 
 The Privacy Pass architecture consists of four logical entities --
 Client, Origin, Issuer, and Attester -- that work in concert as
-shown in {{introduction}} for token issuance and redemption. This
-section describes the purpose of token issuance and redemption
-and the requirements therein on the relevant participants.
+for token issuance and redemption. This section describes the purpose
+of token issuance and redemption and the requirements on the relevant
+participants.
+
+The typical interaction flow for Privacy Pass tokens uses the
+following steps:
+
+1. A Client interacts with an Origin by sending an HTTP request.
+The Origin sends an HTTP response that contains a token challenge
+that indicates a specific Issuer to use.
+Note that the request might be made as part of accessing a
+resource normally, or with the specific intent of triggering a token
+challenge.
```

Comment:
```suggestion
resource normally, or with the specific intent of triggering a Token
Challenge.
```

If we're introducing Token Request/Response here (to be defined in the issuance doc), do we also want to introduce Token Challenge and Token?

Reply:
We do in the diagram, but here I was using it in a bit more colloquial sense.